EIPS/eip-3009.md

```diff
@@ -0,0 +1,317 @@
+---
+eip: 3009
+title: `transferWithAuthorization` - Gas-Abstracted ERC20 transactions with EIP-712
```

Comment:
```suggestion
title: Transfer With Authorization
```
Title should be very short, something you can refer to as part of a sentence.  The rest of this title seems like a good fit for simple summary.

Reply:
Followed your suggestion, thank you.